config/routes.rb

```diff
@@ -8,7 +8,11 @@
   get "githook/master" => "githook#master"
 
   # Resources on the page
-  get '/vecktorn', to: redirect('http://old.fsektionen.se/vecktorn/signup.php'), as: :vecktorn_signup
+  get '/vecktorn', to: redirect('http://eepurl.com/bxH7lv'), as: :vecktorn_signup
```

Comment:
Eep? I would prefer linking to the actual page if possible. This would generate ~4 levels of redirect, hurting SEO among other things.